src/include/storage/index/b_plus_tree.h

```diff
@@ -14,6 +14,7 @@
 #include <string>
 #include <vector>
 
+#include "common/logger.h"
```

Comment:
I'm not very enthusiastic about including logger in a header file because that probably means it'll get included everywhere. But I guess reducing compilation time can be a future problem.

Reply:
Hi, I move the implemention and include of logger.h to b_plus_tree.cpp, ptal.